### PR TITLE
[ref] Use olekukonko/tablewriter (initial pass)

### PR DESCRIPTION
**Pull request by elithrar**
* Use an improved table-writing library to simplify how we output tables - https://github.com/olekukonko/tablewriter
* Only for the user-agent commands (to start with)

```
~ flarectl ua l --zone=questionable.services 
+----------------------------------+--------------------------------+--------------+--------------------------------+-------+
|                ID                |          DESCRIPTION           |     MODE     |             VALUE              | PAUSED  |
+----------------------------------+--------------------------------+--------------+--------------------------------+-------+
| a23b50de3c064a5a860e8b84cd2b382c | Challenge Chrome v61           | challenge    | Mozilla/5.0 (Macintosh         | false |
|                                  |                                |              | Intel Mac OS X 10_12_5)        |       |
|                                  |                                |              | AppleWebKit/537.36 (KHTML like |       |
|                                  |                                |              | Gecko) Chrome/61.0.3163.100    |       |
|                                  |                                |              | Safari/537.                    |       |
| 63eaffa2e76f439facdc6380e6ea110b | Challenge Chrome v61           | challenge    | Mozilla/5.0 (Macintosh         | false |
|                                  |                                |              | Intel Mac OS X 10_12_5)        |       |
|                                  |                                |              | AppleWebKit/537.36 (KHTML like |       |
|                                  |                                |              | Gecko) Chrome/61.0.3163.100    |       |
|                                  |                                |              | Safari/537.3                   |       |
| 2aab5ba4d8f74f82a935124fa450abfc | Challenge Chrome v61           | challenge    | Mozilla/5.0 (Macintosh         | false |
|                                  |                                |              | Intel Mac OS X 10_12_5)        |       |
|                                  |                                |              | AppleWebKit/537.36 (KHTML like |       |
|                                  |                                |              | Gecko) Chrome/61.0.3163.100    |       |
|                                  |                                |              | Safari/537.36                  |       |
| 2487b992f18a4547a39d9d8d2af33e84 | Block the worst browser in the | block        | Wawoo Browser 90310            | false |
|                                  | world                          |              |                                |       |
| 62d49d9eacca445b9973f583688b74e7 |                                | challenge    | Wawoo Browser 90210            | false |
| 1ab4054cca55471387e407b35c6cc267 | block IE6                      | block        | Mozilla/6.0 (osh; Intel Mac OS | false |
|                                  |                                |              | X 10_12_5) AppleWebKit/603.2.4 |       |
|                                  |                                |              | (KHTML, like Gecko)            |       |
|                                  |                                |              | Version/10.1.1 Safari/603.2.4  |       |
| 95ccf4e26e714d7c8e55d9a858258061 | block IE6                      | block        | Mozilla/5.0 (osh; Intel Mac OS | false |
|                                  |                                |              | X 10_12_5) AppleWebKit/603.2.4 |       |
|                                  |                                |              | (KHTML, like Gecko)            |       |
|                                  |                                |              | Version/10.1.1 Safari/603.2.4  |       |
| fdff66612b0a4d73ae72a0f13e7c6693 | block IE6                      | js_challenge | Mozilla/5.0 (Macintosh;        | false |
|                                  |                                |              | Intel Mac OS X 10_12_5)        |       |
|                                  |                                |              | AppleWebKit/603.2.4 (KHTML,    |       |
|                                  |                                |              | like Gecko) Version/10.1.1     |       |
|                                  |                                |              | Safari/603.2.4                 |       |
+----------------------------------+--------------------------------+--------------+--------------------------------+-------+
```

Next steps: expand this to the rest of the CLI.

### CODE REVIEW

**Comment by jamesog** [comment]
I must admit I'm not hugely keen on having the "cell borders", which was why I'd written the simplistic thing originally. I guess that's personal preference though. Willing to leave that open for others throw in their 2c.

Using this library is probably the sensible thing though - I've used it for other internal tools.

**Comment by elithrar** [comment]
Happy to strip the borders too - that's easy enough! 

I will also use this to support multiple output options—namely CSV and JSON—from the CLI.

File: cmd/flarectl/user_agent.go [hunk 1]
```diff
@@ -149,10 +149,10 @@ func userAgentList(c *cli.Context) {
...
 	}
 
-	output := make([]table, 0, len(resp.Result))
+	output := make([][]string, 0, len(resp.Result))
 	for _, rule := range resp.Result {
 		output = append(output, formatUserAgentRule(rule))
 	}
 
-	makeTable(output, "ID", "Description", "Mode", "Value", "Paused")
+	writeTable(output, "ID", "Description", "Mode", "Value", "YOLO")
```
Comment (jamesog) [inline cmd/flarectl/user_agent.go]: Err, was this intentional? :-)

**Comment by elithrar** [comment]
```
                 ID                |          DESCRIPTION           |     MODE     |             VALUE              | YOLO   
+----------------------------------+--------------------------------+--------------+--------------------------------+-------+
  a23b50de3c064a5a860e8b84cd2b382c | Challenge Chrome v61           | challenge    | Mozilla/5.0 (Macintosh         | false  
                                   |                                |              | Intel Mac OS X 10_12_5)        |        
                                   |                                |              | AppleWebKit/537.36 (KHTML like |        
                                   |                                |              | Gecko) Chrome/61.0.3163.100    |        
                                   |                                |              | Safari/537.                    |        
  63eaffa2e76f439facdc6380e6ea110b | Challenge Chrome v61           | challenge    | Mozilla/5.0 (Macintosh         | false  
                                   |                                |              | Intel Mac OS X 10_12_5)        |        
                                   |                                |              | AppleWebKit/537.36 (KHTML like |        
                                   |                                |              | Gecko) Chrome/61.0.3163.100    |        
                                   |                                |              | Safari/537.3                   |        
  2aab5ba4d8f74f82a935124fa450abfc | Challenge Chrome v61           | challenge    | Mozilla/5.0 (Macintosh         | false  
                                   |                                |              | Intel Mac OS X 10_12_5)        |        
                                   |                                |              | AppleWebKit/537.36 (KHTML like |        
                                   |                                |              | Gecko) Chrome/61.0.3163.100    |        
                                   |                                |              | Safari/537.36                  |        
  2487b992f18a4547a39d9d8d2af33e84 | Block the worst browser in the | block        | Wawoo Browser 90310            | false  
                                   | world                          |              |                                |        
  62d49d9eacca445b9973f583688b74e7 |                                | challenge    | Wawoo Browser 90210            | false  
  1ab4054cca55471387e407b35c6cc267 | block IE6                      | block        | Mozilla/6.0 (osh; Intel Mac OS | false  
                                   |                                |              | X 10_12_5) AppleWebKit/603.2.4 |        
                                   |                                |              | (KHTML, like Gecko)            |        
                                   |                                |              | Version/10.1.1 Safari/603.2.4  |        
  95ccf4e26e714d7c8e55d9a858258061 | block IE6                      | block        | Mozilla/5.0 (osh; Intel Mac OS | false  
                                   |                                |              | X 10_12_5) AppleWebKit/603.2.4 |        
                                   |                                |              | (KHTML, like Gecko)            |        
                                   |                                |              | Version/10.1.1 Safari/603.2.4  |        
  fdff66612b0a4d73ae72a0f13e7c6693 | block IE6                      | js_challenge | Mozilla/5.0 (Macintosh;        | false  
                                   |                                |              | Intel Mac OS X 10_12_5)        |        
                                   |                                |              | AppleWebKit/603.2.4 (KHTML,    |        
                                   |                                |              | like Gecko) Version/10.1.1     |        
                                   |                                |              | Safari/603.2.4                 |        
```

File: cmd/flarectl/user_agent.go (hunk 1 above)
Comment (elithrar) [reply cmd/flarectl/user_agent.go]: No. Fixing!

**Comment by jamesog** [comment]
Yeah that looks a bit neater. And 👍 for output options.

**Pull request merged**